shared/templates/pam_options/bash.template

```diff
@@ -7,26 +7,26 @@
 declare -a VALUES=()
 declare -a VALUE_NAMES=()
 declare -a ARGS=()
-declare -a NEW_ARS=()
+declare -a NEW_ARGS=()
 
 {{% for arg in ARGUMENTS -%}}
 {{% if arg['variable']|length -%}}
     {{{ bash_instantiate_variables("var_password_pam_" + arg['variable']) -}}}
-VALUES+=${{{ "var_password_pam_" + arg['variable'] }}}
-VALUE_NAMES+={{{ arg['variable'] }}}
+VALUES+=("${{{ "var_password_pam_" + arg['variable'] }}}")
```

Comment:
I have discovered another problem. The VALUES is glued after NEW_ARGS. Build the SLE 15 content and check build/sle15/fixes/bash/accounts_password_pam_pwhistory_remember.sh:

```

var_password_pam_remember='(bash-populate var_password_pam_remember)'
VALUES+=("$var_password_pam_remember")
VALUE_NAMES+=("remember")
ARGS+=("")
NEW_ARGS+=("")VALUES+=("")
VALUE_NAMES+=("")
ARGS+=("use_authtok")
NEW_ARGS+=("use_authtok")
```

Reply:
it still happens, even after your push-force

Reply:
thanks, now it should be OK.